amenity-points.mss

```diff
@@ -852,7 +854,8 @@
     marker-fill: @airtransport;
   }
 
-  [feature = 'aeroway_aerodrome'][zoom >= 10][zoom < 14] {
+  [feature = 'aeroway_aerodrome'][zoom >= 14][way_pixels < @area-too-large-to-label-size-in-pixels],
+  [feature = 'aeroway_aerodrome'][zoom >= 10][way_pixels > 5][way_pixels < @area-too-large-to-label-size-in-pixels] {
```

Comment:
I am not too sure as I failed to find place with many minor airports tagged as an area - almost all are added solely as nodes.